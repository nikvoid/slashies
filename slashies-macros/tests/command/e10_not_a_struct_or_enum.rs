use slashies_macros::Command;

/// An invalid command (not a struct or enum)
#[derive(Command)]
#[name = "BadCommand"]
union BadCommand {
    x: u64,
}

fn main() {}
