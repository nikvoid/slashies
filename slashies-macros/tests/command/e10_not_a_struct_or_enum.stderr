error: Can only derive Command for structs (regular commands) or enums (commands with subcommands)
 --> tests/command/e10_not_a_struct_or_enum.rs:6:7
  |
6 | union BadCommand {
  |       ^^^^^^^^^^
